proposals/0243-character-literals.md

```diff
@@ -0,0 +1,239 @@
+ Single Quoted Character Literals
+
+ * Proposal: [SE-XXXX](0000-single-quoted-character-literals.md)
+ * Authors: [Kelvin Ma (“Taylor Swift”)](https://github.com/kelvin13), [John Holdsworth](https://github.com/johnno1962)
+ * Review manager: [Ben Cohen](https://github.com/airspeedswift)
+ * Status: **Pending third review** 
+ * Implementation: [https://github.com/apple/swift/pull/61477)
+ * Threads: [1](https://forums.swift.org/t/prepitch-character-integer-literals/10442) [2](https://forums.swift.org/t/se-0243-codepoint-and-character-literals/21188)
+
+ ## Introduction
+
+ Swift emphasizes a unicode-correct definition of 
+what constitutes a `Character`, but unlike most 
+common programming languages, Swift does not have a 
+dedicated syntax for `Character` literals. Instead, 
+three overlapping “ExpressibleBy” protocols and 
```

Comment:
don’t need this, enough to just say "... a double quoted string literal 
can take the role of a `String`, `Character`, or 
`Unicode.Scalar` value depending on its content, and 
the expression context."